### PR TITLE
Add categories to JFrog Artifactory commands

### PR DESCRIPTION
**Pull request by sverdlov93**
- [ ] All [tests](https://github.com/jfrog/jfrog-cli/CONTRIBUTING.md#tests) have passed. If this feature is not already covered by the tests, new tests have been added.
- [ ] The pull request is targeting the `dev` branch.
- [ ] The code has been validated to compile successfully by running `go vet ./...`.
- [ ] The code has been formatted properly using `go fmt ./...`.

---

image

### CODE REVIEW

File: artifactory/cli.go [hunk 1]
```diff
@@ -121,6 +121,15 @@ import (
 	"github.com/urfave/cli"
 )
 
+const filesCategory = "Files"
+const buildCategory = "Build Info"
+const repoCategory = "Repository Management"
+const replicCategory = "Replication"
+const permCategory = "Permission Target"
```
Comment (eyalbe4) [inline artifactory/cli.go:128]: Target --> Targets

File: artifactory/cli.go [hunk 2]
```diff
@@ -121,6 +121,15 @@ import (
 	"github.com/urfave/cli"
 )
 
+const filesCategory = "Files"
```
Comment (eyalbe4) [inline artifactory/cli.go:124]: Files --> Files Management

**Review by eyalbe4 — CHANGES_REQUESTED** [review]

File: artifactory/cli.go [hunk 3]
```diff
@@ -121,6 +121,15 @@ import (
...
 )
 
+const filesCategory = "Files Management"
+const buildCategory = "Build Info"
+const repoCategory = "Repository Management"
+const replicCategory = "Replication"
+const permCategory = "Permission Targets"
+const userCategory = "User Management"
+const transferCategory = "Transfer Between Artifactory Instances"
+const otherCategory = "Other"
```
Comment (yahavi) [inline artifactory/cli.go:131]:
```suggestion
const (
	filesCategory    = "Files Management"
	buildCategory    = "Build Info"
	repoCategory     = "Repository Management"
	replicCategory   = "Replication"
	permCategory     = "Permission Targets"
	userCategory     = "User Management"
	transferCategory = "Transfer Between Artifactory Instances"
	otherCategory    = "Other"
)
```

**Review by yahavi — CHANGES_REQUESTED** [review]
Looks good.
Please consider updating https://github.com/jfrog/documentation accordingly.

**Pull request merged**